src/App.js

```diff
@@ -1,10 +1,22 @@
 import React from 'react';
+import { Routes, Route } from 'react-router-dom';
+import Navbar from './pages/Navbar';
+import Home from './pages/Home';
+import Quote from './pages/Quotes';
 import Calculator from './Components/Calculator';
-import './index.css';
 
 class App extends React.PureComponent {
   render() {
-    return (<Calculator />);
+    return (
+      <>
+        <Navbar />
+        <Routes>
+          <Route path="/home" element={<Home />} />
```

Comment:
- [ ] [optional] Instead of making this path to the home you could leave it to be the default page as shown below:
```suggestion
          <Route path="/" element={<Home />} />
```

Reply:
Thanks for the suggestion @danmainah